website_snippet_country_dropdown/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Snippet's dropdown country VAT code
+===================================
+
+This module adds a template with a dropdown and an input text field, is a base
+for be inherited by others modules into a html form.
+
+Usage
+=====
+
+To extends this template you need inherit *country_dropdown* template and adds
+you personal code.
+The template have two input text, you can create your own variable for these
+inputs and pass it to the template with the name *vat_field* and
```

Comment:
We spent a lot of time, and somehow Odoo breaks snippets if you call them from a t-call, so I'm afraid it will be hard to have it as both UI snippet and t-call one.